content/faq/30_what-should-i-expect-from-mentors/index.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I think we now have a more detailed answer to this. It includes how you should raise issue in this repo to ask for help.

We should make this answer more rich. Talk to Sahithi from the volunteers team for this.

Reply:
In my latest FAQ PR i have added a new question (47) -> "_how to make sure that you get a mentor for a review?_" 
Will that question replace this one?
It has a really good and detailed answer.

Reply:
yes.. that is great.. remove this question from here then..